tests/system/action/user/test_create.py

```diff
@@ -824,3 +824,18 @@ def test_create_forbidden_username(self) -> None:
         )
         self.assert_status_code(response, 400)
         assert "This username is forbidden." in response.json["message"]
+
+    def test_create_gender(self) -> None:
+        response = self.request(
+            "user.create",
+            {
+                "username": "test_Xcdfgee",
+                "organization_management_level": OrganizationManagementLevel.CAN_MANAGE_USERS,
```

Comment:
Please remove since it is not relevant to the test.

Reply:
I simplified this test.